api/client/trust.go

```diff
@@ -425,6 +425,10 @@ func (cli *DockerCli) trustedPush(repoInfo *registry.RepositoryInfo, tag string,
 	}
 
 	err = repo.Publish()
+	if err == nil {
+		fmt.Fprintf(cli.out, "Finished signing %q with tag %s\n", repoInfo.FullName(), tag)
```

Comment:
It's weird that this message will only print one tag, but it appears that the code loops to add all targets if they came back from the `push`. I think its because the client won't ask the daemon to push everything unless you specify no tag, but this code doesn't allow the trust metadata push unless a tag is explicitly added.